Lesson Modules/Module 1- Import Local CSVs.ipynb

```diff
@@ -56,12 +56,13 @@
    "cell_type": "code",
    "execution_count": 3,
    "metadata": {
-    "collapsed": true
+    "collapsed": false
    },
    "outputs": [],
    "source": [
-    "def printlocalCSV():\n",
-    "    csvfile = open(filename)             # open the file with Python\n",
+    "def printlocalCSV(filepath, filename):\n",
+    "    file = filepath + filename           # concatenate filepath and filename into one string\n",
+    "    csvfile = open(file)                 # open the file with Python\n",
```

Comment:
Here you use the `open()` function but below in `importlocalCSV` you use the `with` paradigm. `with` is much better in general, although one could argue that it is more complicated for a beginner to understand.
Two options:
1. Keep the `open()` function here. However, you'd better close it: `csvfile.close()` after you're done using it.
2. Use the `with` paradigm here instead of `open` and `close`.
I would suggest option 2. You just want to explain in a comment what `with` does: it opens the file, and whenever you leave that code block it will close the file for you.